src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/DataController.cs

```diff
@@ -325,7 +327,7 @@ public async Task<IActionResult> OverwriteData(int instanceOwnerId, Guid instanc
                     data.FileName = updatedData.FileName;
                     data.LastChangedBy = User.Identity.Name;
                     data.LastChangedDateTime = changedTime;
-
+                    data.Refs = updatedData.Refs;
```

Comment:
If the query param is not used the refs value is will always be set to null. Hence if you add refs in  a post and then do a PUT with no refs, the refs is deleted. Is this the wanted behaviour? I initially thought that refs should be unaltered if I drop the refs param.

Reply:
Good question, and an interesting problem. I feel like we kind of do something "wrong" here either way. If we choose to not do anything if refs are not supplied (value is null), how are you supposed to delete the refs? 
But I certainly get your point, and I'm unsure what the best solution is here.